src/handlers.js

```diff
@@ -51,32 +57,21 @@ const handlers = {
                 .get();
     },
 
-    signUpUserCallback: ({ challongeService, userRepository }) => async function ({ sender, originalRequest }) {
-        const user = await userRepository.getUser(sender);
-        if (!user) {
-            return unknownUserResponse;
-        }
-
-        const { actions, callback_id } = originalRequest;
-        const { value: tournamentId } = R.find(({ name }) => name === 'sign_up')(actions);
-
-        if (!tournamentId) {
-            throw new Error(`Invalid action value(s) for callback: ${callback_id}`);
-        }
-
+    signUpUserCallback: ({ challongeService, userRepository }) => validateCallbackValue('sign_up', validateSendingUser(userRepository, async function (message, userPromise, tournamentId) {
```

Comment:
Could we use `R.pipe` here instead, piping `validateSendingUser` -> `validateCallbackValue` -> actual handler? I assume that we couldn't pass in multiple args like we do now, but rather would extend the message object...

Also, would we be forced to throw errors on validation failure (like for the async `validateSendingUser`?